### ISSUE TITLE
Display publication date on charmhub

### ISSUE THREAD

**Issue by heitorPB** [canonical-web-and-design/charmhub.io#1070]
When we browse the versions/channels of a charm in Charmhub, we can't see when that version was published. It is available with `juju info charmname` only.

It would be nice to have another column in Charmhub showing this info, maybe after the `version` column:

![versions](https://user-images.githubusercontent.com/13461702/124817666-793eec80-df40-11eb-8191-4405d1586e03.png)

```bash

$ juju info slurmd
name: slurmd
charm-id: PNY1jWPbggzG1NZvpftYqLgg86qKyNtd
summary: |
  Slurmd, the compute node daemon of Slurm.
publisher: Omnivector Solutions
supports: focal, centos7
subordinate: false
store-url: https://charmhub.io/slurmd
description: |
  This charm provides slurmd, munged, and the bindings to other utilities
  that make lifecycle operations a breeze.

  slurmd is the compute node daemon of Slurm. It monitors all tasks running
  on the compute node , accepts work (tasks), launches tasks, and kills running tasks upon request.
relations:
  provides:
    slurmd: slurmd
  requires: {}
channels: |
  latest/stable:     0.6.6  2021-07-01  (19)  4MB
  latest/candidate:  0.6.5  2021-06-28  (18)  4MB
  latest/beta:       ↑
  latest/edge:       0.6.6-1-g07b1fc0  2021-07-05  (20)  4MB
```

**Comment by heitorPB** [comment]
The Embeddable card provided in https://charmhub.io/slurmd/publicise/cards shows the publication date:

![slurmd-card-charmhub](https://user-images.githubusercontent.com/13461702/125477589-578bb3ae-042e-4fe9-98b4-3c62580e51d0.png)

**Comment by heitorPB** [comment]
Implemented in #1088 

Thanks!
